build.gradle.kts

```diff
@@ -0,0 +1,201 @@
+import org.ajoberstar.grgit.Grgit
+import com.github.spotbugs.*
+
+buildscript {
+    repositories {
+        jcenter()
+        maven {
+            setUrl("https://github.com/WPIRoboticsProjects/opencv-maven/raw/mvn-repo")
```

Comment:
HTTPS support. Wonderful!

Github might get pissy though if we keep using them as a maven repo.